UI/MapUI.gd

```diff
@@ -5,7 +5,7 @@ var room_ui: PackedScene = load("res://Map/RoomUI.tscn")
 var _padding_offset = 20
 
 func _input(_inputevent: InputEvent) -> void:
-	if Input.is_key_pressed(KEY_ESCAPE):
+	if (_inputevent is InputEventKey and _inputevent.pressed and _inputevent.keycode == KEY_ESCAPE):
```

Comment:
can you use the [InputMap](https://docs.godotengine.org/en/stable/classes/class_inputmap.html) instead of having a `KEY_ESCAPE` ?